docs/conf.py

```diff
@@ -12,10 +12,13 @@
 import sys
 import inspect
 import shutil
+from pkg_resources import get_distribution
 
-__location__ = os.path.join(
-    os.getcwd(), os.path.dirname(inspect.getfile(inspect.currentframe()))
-)
+# Get version information for the documentation
```

Comment:
Kannst du spezifizieren für welche Dokumentation?

Reply:
40dda41  
Habe noch einen Link zu unserer read the docs docu hinzugefügt